tealiumlibrary/src/test/java/com/tealium/core/consent/ConsentManagerTest.kt

```diff
@@ -2,18 +2,24 @@ package com.tealium.core.consent
 
 import android.app.Application
 import android.content.SharedPreferences
+import android.util.Log
```

Comment:
`import android.util.Log`, `import com.tealium.core.Logger` and `import com.tealium.core.messaging.EventDispatcher` are all unused. can remove.